utility_analysis_new/tests/utility_analysis_test.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility Analysis Public API Test"""
+import unittest
+
+import pipeline_dp
+from utility_analysis_new import utility_analysis
+
+
+class UtilityAnalysis(unittest.TestCase):
+
+    def _get_default_extractors(self) -> pipeline_dp.DataExtractors:
+        return pipeline_dp.DataExtractors(
+            privacy_id_extractor=lambda x: x,
+            partition_extractor=lambda x: x,
+            value_extractor=lambda x: x,
+        )
+
+    def test_utility_analysis(self):
```

Comment:
Nit: `test_wo_public_partitions` 
in order to differentiate from the other test